lib/widgets/story_view.dart

```diff
@@ -169,18 +171,30 @@ class StoryItem {
     bool shown = false,
     bool roundedTop = true,
     bool roundedBottom = false,
+    double roundedBottomRadius = 8.0,
+    double roundedTopRadius = 8.0,
     Widget? loadingWidget,
     Widget? errorWidget,
     EdgeInsetsGeometry? captionOuterPadding,
     Duration? duration,
   }) {
     return StoryItem(
-      ClipRRect(
```

Comment:
Please there is a reason ClipRRect is there.
please add it back and implement your solution around it borderRadius: